src/bproxy.c

```diff
@@ -257,6 +259,7 @@ void proxy_read_cb(uv_stream_t *handle, ssize_t nread, const uv_buf_t *buf)
     {
       log_error("could not read from socket! (%s)", uv_strerror(nread));
     }
+    // TODO: Fix premature closing of client connection (in case of pending writes)
```

Comment:
why TODO mark? it would be nice if we can make a release soon.

Reply:
I'll fix it today... I just didn't want to forget it.